packages/cashier/src/stores/withdraw-store.ts

```diff
@@ -1,30 +1,34 @@
-import React from 'react';
+import { ReactElement } from 'react';
```

Comment:
We have the TReactElement type in the types folder and I think it's better to use this type for consistency across other components.